tools/codegen/gen_functionalization_type.py

```diff
@@ -40,15 +40,22 @@ def is_tensor_like(a: Union[Argument, TensorOptionsArguments, SelfArgument]) ->
 # unwraps all tensor-like arguments, returning:
 # (1) a string containing all of the logic that does the unwrapping
 # (2) a context, to be used by translate(), with all of the relevant bindings.
-def unwrap_tensor_args(sig: DispatcherSignature) -> Tuple[str, List[Binding]]:
+def unwrap_tensor_args(sig: DispatcherSignature, *, include_sync: bool) -> Tuple[str, List[Binding]]:
```

Comment:
How should I know to call this with `include_sync=True` versus False?

Reply:
So it looks like the distinction here at the call site is "is it a view or not". I'd suggest naming the parameter after *that*, makes it very easy to understand how you should call it, and then operational consequences of syncing or not are localized to this function.